api/v1/prefixclaim_types.go

```diff
@@ -120,3 +130,17 @@ var ConditionPrefixAssignedFalse = metav1.Condition{
 	Reason:  "PrefixCRNotCreated",
 	Message: "Failed to fetch new Prefix from NetBox",
 }
+
+var ConditionParentPrefixComputedTrue = metav1.Condition{
```

Comment:
To match with the wording of `ParentPrefixSelector`, could we maybe use `Selected` instead of `Computed`?

Reply:
All the `computed` occurrences are replaced with `selected`